### ISSUE TITLE
there seems to be a new tool for vfb to ufo conversion 

### ISSUE THREAD

**Issue by typemytype** [robotools/extractor#37]
https://pypi.org/project/vfb2ufo3/

**Comment by typemytype** [comment]
Would be more then happy with a PR for Xmas!! :)

**Comment by jenskutilek** [comment]
Just found this old issue ... vfbLib can extract a UFO on it's own, would you still like to see this functionality also provided by extractor?

**Comment by benkiel** [comment]
Seeing that `vfb3ufo` is doing a much better job of VFB to UFO, maybe we should either take out VFB support or have a dependency on `vfb3ufo` for VFB support (https://github.com/robotools/extractor/pull/68#discussion_r1747623887)

**Comment by benkiel** [comment]
Yes, more irrelevant, but I hesitate to take it out as folks may depend on it. @typemytype ?

**Comment by jenskutilek** [comment]
It's pretty easy to swap `vfb2ufo` with vfbLib: #70 :)

**Comment by rimas-kudelis** [comment]
Should this be closed in favor of #70?

**Comment by benkiel** [comment]
Yes, closing.
